src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -10,12 +10,12 @@
 
   <ItemGroup>
     <None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
-    <Compile Include="..\OSSkipConditionFactAttribute.cs" LinkBase="Shared" />
```

Comment:
This seems related to the build failure. I'm guessing it was removed because of the new project reference? Perhaps instead of referencing a test project from another, whatever is needed from it could be move to the common project?

Reply:
I've undone the move of the TagHelperServiceTestBase in this PR which caused a lot of these test changes. We can refactor later if needed. This is no longer removed here.